src/handlers/comment/handlers/multiplier.ts

```diff
@@ -18,13 +18,32 @@ export const multiplier = async (body: string) => {
     return;
   }
 
-  const regex = /\/multiplier @(\w+) (\d+(?:\.\d+)?)/; // /multiplier @0xcodercrane 0.5
+  const regex = /(".*?"|[^"\s]+)(?=\s*|\s*$)/g;
+  // possible commands
+  // /multiplier @user 0.5 "Multiplier reason"
+  // /multiplier 0.5 @user "Multiplier reason"
+  // /multiplier "Multiplier reason" @user 0.5
+  // /multiplier 0.5 "Multiplier reason" @user
+  // /multiplier @user "Multiplier reason" 0.5
```

Comment:
```suggestion
/** You can use this command to set a multiplier for a user.
 * It will accept arguments in any order.
 * Example usage:
 *
 * /multiplier @user 0.5 "Multiplier reason"
 * /multiplier 0.5 @user "Multiplier reason"
 * /multiplier "Multiplier reason" @user 0.5
 * /multiplier 0.5 "Multiplier reason" @user
 * /multiplier @user "Multiplier reason" 0.5
 * 
 **/
```

Reply:
Done